src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
```

Comment:
The body hasn't a padding
```suggestion
```

Reply:
Still not fixed.

Reply:
<img width="648" alt="image" src="https://github.com/user-attachments/assets/5892519b-06ba-4925-a149-f14b4ab89112">